app/web/features/communities/events/LongEventCard.tsx

```diff
@@ -104,58 +135,70 @@ const LongEventCard = ({
   const isCancelled = event.isCancelled;
 
   return (
-    <Card className={classes.root} data-testid="event-item">
-      <Link
-        href={routeToEvent(event.eventId, event.slug)}
-        className={classes.card}
-      >
-        <CardMedia
-          className={classes.cardMedia}
-          component="img"
-          image={event.photoUrl || eventImagePlaceholderUrl}
-        />
-        <CardContent className={classes.cardContent}>
-          <div className={classes.row}>
-            <Tooltip title={event.title}>
-              <Typography variant="h3" className={classes.title}>
-                {event.title}
-              </Typography>
-            </Tooltip>
-            <div className={classes.tags}>
-              {isCreatedByMe && (
-                <Pill variant="rounded">{t("communities:created_by_me")}</Pill>
-              )}
-              {isOnline && (
-                <Pill variant="rounded">{t("communities:online")}</Pill>
-              )}
-              {isCancelled && (
-                <Pill
-                  backgroundColor={theme.palette.error.main}
-                  color={theme.palette.common.white}
-                  variant="rounded"
-                >
-                  {t("communities:cancelled")}
-                </Pill>
-              )}
-            </div>
-          </div>
-          <div className={classes.row}>
-            <div className={classes.eventInfo}>
-              {event.offlineInformation
-                ? event.offlineInformation.address
-                : t("communities:virtual_event_location_placeholder")}
-
-              <div>{startTime}</div>
-            </div>
-            <div className={classes.attendees}>
-              {t("communities:attendees_count", {
-                count: event.goingCount + event.maybeCount,
+    <StyledCard data-testid="event-item">
+      <Link href={routeToEvent(event.eventId, event.slug)} passHref>
+        <CardLayout>
+          <ImageWrapper>
+            <CardMedia
+              component="img"
+              image={event.photoUrl || eventImagePlaceholderUrl}
+              title={event.title}
+              sx={(theme) => ({
+                height: "100%",
+                width: "100%",
+                objectFit: "cover",
+                [theme.breakpoints.down("sm")]: {
+                  height: theme.spacing(25),
+                },
               })}
-            </div>
-          </div>
-        </CardContent>
+            />
+            <FlagWrapper>
+              <FlagButton
+                contentRef={`event/${event.eventId}`}
+                authorUser={event.creatorUserId}
+              />
+            </FlagWrapper>
+          </ImageWrapper>
+
```

Comment:
Okay I think this image wrapper is the culprit here. Remove the `ImageWrapper` added element. Then grab the `cardMedia` style from the former `useStyles` and put it in the `sx` of `CardMedia` intstead. That should fix the stretched image.